README.md

```diff
@@ -408,7 +408,7 @@ See `example/deepview` for actual usage.
 
 ## Defines
 
-* TINYEXR_USE_MINIZ  Set `1`(default) to use embedded miniz compression. If you want zlib or custom zlib compatible library, set `0` and link zlib library.
+* TINYEXR_USE_MINIZ  Set `1` (default) to use embedded miniz compression. If you want zlib or custom zlib compatible library, set `0` and link zlib library.
```

Comment:
BTW, it seems like this section is outdated documentation? It doesn't match `### Compile flags` earlier and seems redundant. The `## TODO` section also seems to overlap with the earlier `Current status` section.

Reply:
Good catch. Yes we should remove this `## Defines` section.

Reply:
Done.

Reply:
:pray: